creator-node/src/config.js

```diff
@@ -183,6 +183,12 @@ const config = convict({
     env: 'rateLimitingTrackReqLimit',
     default: null
   },
+  URSMRequestForSignatureReqLimit: {
+    doc: 'Total requests per hour rate limit for /ursm_request_for_signature route',
+    format: 'nat',
+    env: 'URSMRequestForSignatureReqLimit',
+    default: null
```

Comment:
should this be defaulted to null?

Reply:
just went with the established pattern for the rate limit configs, which are all defaulted to null

Reply:
yeh could have done it that way, would have maybe been better, just stuck with current pattern without too much thought tbh